app/components/post_body_additional_content/post_body_additional_content.js

```diff
@@ -377,8 +381,42 @@ export default class PostBodyAdditionalContent extends ImageViewPort {
         return null;
     };
 
+    renderAppEmbeds = () => {
+        const {
+            post,
+            postProps,
+            baseTextStyle,
+            blockStyles,
+            deviceHeight,
+            deviceWidth,
+            onPermalinkPress,
+            textStyles,
+            theme,
+        } = this.props;
+        const {app_bindings} = postProps;
+
+        if (app_bindings && app_bindings.length) {
+            return (
+                <EmbeddedBindings
+                    embed={app_bindings}
+                    baseTextStyle={baseTextStyle}
+                    blockStyles={blockStyles}
+                    deviceHeight={deviceHeight}
+                    deviceWidth={deviceWidth}
+                    post={post}
```

Comment:
Since there are no connectors on EmbeddedBindings yet, I tried not to add any. But yes, I can add a connector and get the post at EmbeddedBinding level.

Reply:
After reviewing this, I saw that the only place we need the post is down in the actions, and those already had the connector file in place. Got the post there.